Zend/zend_vm_def.h

```diff
@@ -8666,7 +8666,7 @@ ZEND_VM_HANDLER(182, ZEND_BIND_LEXICAL, TMP, CV, REF)
 		}
 	} else {
 		var = GET_OP2_ZVAL_PTR_UNDEF(BP_VAR_R);
-		if (UNEXPECTED(Z_ISUNDEF_P(var)) && !(opline->extended_value & ZEND_BIND_IMPLICIT)) {
+		if (UNEXPECTED(Z_ISUNDEF_P(var)) && !(opline->extended_value & (ZEND_BIND_IMPLICIT|ZEND_BIND_EXPLICIT))) {
```

Comment:
Doesn't this change suppress the undef var notice for `function() use($undef) {}`?

Reply:
It does not, but nor is it necessary, this is BIND_LEXICAL ... over eager search and replace :)